types/setting.go

```diff
@@ -195,6 +199,28 @@ var (
 		SettingNameBackingImageCleanupWaitInterval:              SettingDefinitionBackingImageCleanupWaitInterval,
 		SettingNameGuaranteedEngineManagerCPU:                   SettingDefinitionGuaranteedEngineManagerCPU,
 		SettingNameGuaranteedReplicaManagerCPU:                  SettingDefinitionGuaranteedReplicaManagerCPU,
+		SettingNameSupportBundleImage:                           SettingDefinitionBundleImage,
+		SettingNameSupportBundleImagePullPolicy:                 SettingDefinitionSupportBundleImagePullPolicy,
```

Comment:
We shouldn't need a pull policy (see other images for example). It should be `IfNotPresent` by default for the air gap environment.

Reply:
Updated to use the existing image pull policy: SettingDefinitionSystemManagedPodsImagePullPolicy